contracts/Inbox.sol

```diff
@@ -211,12 +225,22 @@ contract Inbox is IInbox, Eco7683DestinationSettler, Ownable, Semver {
     ) external payable returns (bytes[] memory) {
         emit AddToBatch(_expectedHash, _route.source, _claimant, _prover);
 
-        bytes[] memory results = _fulfill(
+        (bytes[] memory results, uint256 remainingValue) = _fulfill(
             _route,
             _rewardHash,
             _claimant,
             _expectedHash
         );
+        
+        require(
+            remainingValue >= minBatcherReward,
```

Comment:
@nknavkal maybe could just use `address(this).balance` here and remove `remainingValue` calculation from `_fulfill`?